### DISCUSSION TITLE
Apply multiple animations to an element

### DISCUSSION THREAD

**Discussion by marrobHD** [tailwindlabs/tailwindcss#12393] [Help] [answered]
Hi,

i'd like to know if there is a way to apply multiple animations (starting after each other) to an element.

With the following config, only the last defined animation will play. 
I added the following in `tailwind.config.ts`:
```ts
  theme: {
    extend: {
      animation: {
        sparkle: 'extend 1s 1, rotate 1s infinite',
      },
      keyframes: {
        extend: {
          'from': { transform: 'scale(1) '},
          'to': { transform: 'scale(1.5) ' },
        },
        rotate: {
          '0%, 100%': { transform: 'rotate(-15deg) scale(1.5) '},
          '50%': { transform: 'rotate(15deg) scale(1.5) ' },
        },
    },
  }
```

Added the following to the `.tsx`
```tsx
                <div className="animate-sparkle">
                    <p className="">🍄</p>
               </div>
```

**Comment by wongjn** [comment, marked as answer]
Consider having a delay on the second animation that is equal to the duration of the animation before it:

```diff
 extend: {
   animation: {
-    sparkle: 'extend 1s 1, rotate 1s infinite',
+    sparkle: 'extend 1s 1, rotate 1s 1s infinite',
   },
   keyframes: {
```
https://play.tailwindcss.com/dU9mtYisre?file=config

**Reply by marrobHD** [reply]
Thanks!

**Comment by alex-krasikau** [comment]
Hey @marrobHD!

You can check out [tailwindcss-animate](https://github.com/jamiebuilds/tailwindcss-animate) plugin. It provides simple utility classes you can use to set up more complex animations.
